lib/Test/Nginx/Socket.pm

```diff
@@ -2686,6 +2711,24 @@ will produce the following line (to C<stderr>) while running this test block:
 You need to remember to set the C<TEST_NGINX_NO_CLEAN> environment to 1 to prevent the nginx
 and other processes from quitting automatically upon test exits.
 
+=head2 response_body_filters
+
+Transform value for the body of the response_body by the a chain of filters.
```

Comment:
This is still ambiguous. We need to make it clear that we apply the filters to the *actual* response body data instead of the expected response body data.